tests/test_views_schema_collection.py

```diff
@@ -0,0 +1,81 @@
+from kinto.core.testing import unittest
+
+from .support import BaseWebTest
+
+
+BUCKET_URL = '/buckets/blog'
+COLLECTION_URL = '/buckets/blog/collections/articles'
+
+
+SCHEMA = {
+    'title': 'Collection schema',
+    'type': 'object',
+    'required': ['displayFields'],
+    'properties': {
+        'uiSchema': {
+            'type': 'object'
+        },
+        'displayFields': {
+            'type': 'array',
+            'items': {
+                'type': 'string'
+            },
+        },
+    },
+}
+
+VALID_COLLECTION = {'uiSchema': {'schema': {'ui:widget': 'hidden'}}, 'displayFields': ['name']}
+
+
+class DeactivatedSchemaTest(BaseWebTest, unittest.TestCase):
```

Comment:
It's kind of unfortunate that the schema validation feature doesn't have to be turned on here, because it claims `collection:schema` as a reserved word..

Reply:
What do you mean?

That it's a pity to reserve the field `collection:schema` when the feature is disabled?

Reply:
Yes, that.